package/submariner-route-agent.sh

```diff
@@ -9,4 +9,18 @@ else
     DEBUG="-v=4"
 fi
 
-exec submariner-route-agent ${DEBUG} -alsologtostderr
\ No newline at end of file
+# if host is mounted on /host and host has it's own iptables version
+# use that one instead via the shipped iptables wrapper, we do this
+# to avoid issues with hosts
```

Comment:
Instead of "to avoid issues with hosts", shall we use "to avoid configuring iptables and nftables on the host which could lead to functional failures"

Reply:
Thanks, good comment. I extended the explanation a little bit.